src/modules/75-cf/components/FlagActivation/__tests__/JiraIssueList.test.tsx

```diff
@@ -48,6 +78,156 @@ describe('JiraIssueList', () => {
     expect(screen.getByText('ISSUE_2')).toHaveAttribute('href', 'https://jira.com/ISSUE_2')
   })
 
+  test('it should display "Add Jira Issue Modal" on button click', async () => {
+    renderComponent()
+
+    const jiraModalButton = screen.getByRole('button', { name: 'cf.featureFlags.jira.newJiraIssueButton' })
+    expect(jiraModalButton).toBeInTheDocument()
+    userEvent.click(jiraModalButton)
+
+    //assert dialog elements appear
+    expect(screen.getByText('cf.featureFlags.jira.jiraModalTitle')).toBeInTheDocument()
+    expect(screen.getByPlaceholderText('- cf.featureFlags.jira.inputPlaceholder -')).toBeInTheDocument()
+    expect(screen.getByText('add')).toBeInTheDocument()
+    expect(screen.getByText('cancel')).toBeInTheDocument()
```

Comment:
Would be cool to use `getByRole` for these
```suggestion
    expect(screen.getByRole('button', { name: 'add' })).toBeInTheDocument()
    expect(screen.getByRole('button', { name: 'cancel' })).toBeInTheDocument()
```